### PR TITLE
Added sendCharacter

### PR DESCRIPTION
**Pull request by PikAleex**
Wrote code in WinMain to open the port like we did assignment 1. And wrote a separate function to send the character 'a'.

### CODE REVIEW

File: Final_Project/Main.cpp [hunk 1]
```diff
@@ -97,6 +99,18 @@ int WINAPI WinMain(HINSTANCE hInst, HINSTANCE hprevInstance,
 	//set LAST_EOT_RECEIVED to current time
 	LAST_EOT_RECEIVED = time(0);
 
+	//open com port
+	if ((portHandle = CreateFile(lpszCommName, GENERIC_READ | GENERIC_WRITE, 0,
+		NULL, OPEN_EXISTING, 0, NULL))
+		== INVALID_HANDLE_VALUE)
+	{
+		MessageBox(NULL, TEXT("Error opening COM port:"), TEXT(""), MB_OK);
+		return FALSE;
```
Comment (dstrigoun) [inline Final_Project/Main.cpp]: I don't think there's any need to return false here. Maybe we can exit the program?

File: Final_Project/Main.cpp [hunk 2]
```diff
@@ -266,4 +282,29 @@ void terminateProgram()
...
+--
+--	DATE:			November 23, 2018
+--
+--	REVISIONS:		November 23, 2018
+--
+--	DESIGNER:		Dasha Strigoun, Kieran Lee, Alexander Song, Jason Kim
+--
+--	PROGRAMMER:		Alexander Song
+--
+--	INTERFACE:		void sencCharacter(HWND hwnd)
```
Comment (dstrigoun) [inline Final_Project/Main.cpp]: This is very small, but there's a typo in sendCharacter.

File: Final_Project/Main.h [hunk 3]
```diff
@@ -9,6 +9,8 @@ static char Name[] = "GTID";
 static DWORD MAX_RANDOM_WAIT_TIME_MS = 500;
 static DWORD IDLE_TIMEOUT_TIME_S = 30;
 static DWORD CHECK_IDLE_TIMEOUT_MS = 5000;
+LPCSTR lpszCommName = "com1";
+char str[80] = "";
```
Comment (dstrigoun) [inline Final_Project/Main.h]: Correct me if I'm wrong, but I think only static values should go in the header file. Being that `lpszCommName` should never change I think we can make that static, but I think `str` should probably go back in the main file.

Comment (PikAleex) [reply Final_Project/Main.h]: Will change, I was kind of shooting in the dark on where to put them

File: Final_Project/Main.cpp (hunk 2 above)
Comment (PikAleex) [reply Final_Project/Main.cpp]: thanks for catching

File: Final_Project/Main.cpp (hunk 1 above)
Comment (PikAleex) [reply Final_Project/Main.cpp]: that would make sense

**Review by dstrigoun — APPROVED** [review]
Looks good to me!! Thanks alex!

**Pull request merged**